app/locale/en.coffee

```diff
@@ -963,7 +964,7 @@
     average_levels: "Average levels completed:"
     total_levels: "Total levels completed:"
     furthest_level: "Furthest level completed:"
-    concepts_covered: "Concepts Covered"
+    # concepts_covered: "Concepts Covered"
```

Comment:
Do text surgery in all the locale files yourself. I can do this with Emacs in like 2 minutes if you need–just let me know the keys to move from where to where.